api/tests/unit/features/versioning/test_unit_versioning_views.py

```diff
@@ -1639,8 +1639,7 @@ def test_list_versions_always_returns_current_version_even_if_outside_limit(
 
 
 @pytest.mark.freeze_time(now - timedelta(days=DEFAULT_VERSION_LIMIT_DAYS + 1))
-@pytest.mark.parametrize("is_saas", (True, False))
-def test_list_versions_returns_all_versions_for_enterprise_plan(
```

Comment:
Aren't we losing a test here? I understood that we were going to split the test into 2 individual tests? 🤔

Reply:
It was moved into the licensing repo over [here](https://github.com/Flagsmith/licensing/blob/main/tests/integration/test_integration_versioning_views.py#L29) since it needed the `OrganisationLicence` model.